src/plugins/janus_streaming.c

```diff
@@ -4516,7 +4652,9 @@ static json_t *janus_streaming_process_synchronous_request(janus_streaming_sessi
 			}
 			mp->viewers = g_list_remove_all(mp->viewers, s);
 			viewer = g_list_first(mp->viewers);
-			janus_mutex_unlock(&s->mutex);
+			janus_mutex_unlock(&session->mutex);
```

Comment:
This should be `s`, not `session`. I fixed this in a commit recently.

Reply:
Good catch! Fixed.